### ISSUE TITLE
For contributors (для участников)

### ISSUE THREAD

**Issue by JeanPaulLucien** [JeanPaulLucien/New-Android-for-Wexler-Tab-7t#1]
If you wish to help with development, read information and join. Post a message to get additional info, assistance or help.

Если вы желаете помочь в разработке, ознакомьтесь с информацией и присоединяйтесь. Напишите сообщение, чтобы получить дополнительную информацию, содействие или помощь.

Для участия, непосредственно в разработке вам потребуется выполнить инструкции: https://www.unlegacy-android.org/projects/unlegacy-android/wiki/Sync_ua - обращаю внимание, что рекомендуется процессор 4 ядра, 4 ГБ ОЗУ. Всё это ставится на компьютер, не на планшет.

Переводчик: https://translate.google.com

**Comment by clamor-s** [comment]
Are you still interested in development for Wexler Tab?

**Comment by JeanPaulLucien** [comment]
@clamor95, good afternoon, nice to see you again. yeah, I completed https://www.unlegacy-android.org/projects/unlegacy-android/wiki/Sync_ua to step 2. Compile. I have no Telegram account, I tried to register, but was banned in the moment. I sent a message to support, but they didn't answered.

**Comment by clamor-s** [comment]
Try once more time to register. Russian users may have problems with Telegram unfortunately.

**Comment by gyhuyjioikyujh** [comment]
правильно ли я понял: что следует установить убунту через линуксдеплой в планшете и уже в ней компилировать андройд по инструкции подключив 200гб флешку\жесткий диск? и какой именно запустится 4.4, 5, 6, 7?

**Comment by clamor95** [comment]
No, rom is build on PC

**Comment by gyhuyjioikyujh** [comment]
Is it possible to build on intel atom525 4core? How to download to device if it build's?

**Comment by clamor95** [comment]
Wrong. Your questions were wrong. Right question is: "Is wexler tab already supported?"

**Comment by JeanPaulLucien** [comment]
Пока собирал информацию по железу устройства, натолкнулся на репозиторий:
https://android.googlesource.com/kernel/tegra/+refs
Попал я туда по ссылке: https://android.googlesource.com/kernel/tegra/+/android-tegra-flounder-3.10-lollipop-release/drivers/mfd/tlv320aic3256-i2c.c - что, как я понимаю, является драйвером на звук.

**Comment by clamor-s** [comment]
You need to update board first. Drivers will be after kernel boots.

**Comment by DmitryADP** [comment]
Вот это надо переделать под новое ядро https://github.com/DmitryADP/diff_qc750/tree/master/kernel/arch/arm/mach-tegra/board-nabi2...
Последний раз когда собирал на них и застрял)

**Comment by clamor-s** [comment]
@DmitryADP @JeanPaulLucien Board is [ported](https://github.com/JeanPaulLucien/android_kernel_nvidia_tegra3/commit/af57d3e70872d9ea03ad10ef9f6072fd11bbe664) already. You have to update it to state of 3.4 using as a gide this portings:

* [Acers](https://github.com/Unlegacy-Android/android_kernel_nvidia_tegra3/commits/stable?after=680963c31067155706ced738b7be5ad01f9c49d5+419);
* [Transformers](https://github.com/clamor95/android_kernel_nvidia_tegra3/commits/board-asus-exp?after=6f66a193f3098d5e4cfbed17a10ebe84ad687056+209);
* [Lenovo IdeaTab](https://github.com/PJBrs/android_kernel_lenovo_kai/commits/A2109-linux-tegra-nv-3.4-unlegacy?after=e41c4c7f94343b85b640e9d9a0d2440ba0c95e89+209);

**Comment by DmitryADP** [comment]
Сделал рабочее дерево для 7.1.2. Система на нем собралась. За конфиг ядра пока взят от 2109а.

 https://github.com/DmitryADP/device_qc750

Из минусов-рекавери получается больше стандартного раздела, поэтому в конфиге он увеличен с 8 до 10мб, заодно и бут тоже. Придется переразмечать память устройства-делается достаточно просто с помощью имеющихся файлов прошивки от Векслер. Откорректированный flash.cfg имеется.
Надо ещё точно править всё в  папке sepolicy и proprietary-files.

**Comment by clamor-s** [comment]
@DmitryADP here is [wexler recovery tree](https://github.com/clamor95/android_device_unlegacy_recovery/tree/qc750?files=1). I wouldn't recommend to do repartition until you make working rom and recovery build on your current layout. 

P.S. ~80% of your tree isn't needed. Check how UA Lenovo Idea Pad tree is done

**Comment by DmitryADP** [comment]
Так я на на основе Леново и делал.
Переразметка может и не понадобится, видно будет из размеров ядра.

**Comment by clamor-s** [comment]
@DmitryADP @JeanPaulLucien I can propose you postmarket OS on 5.14.0 kernel if you are interested. I can guide, but dts writing is left to you.
